Purchases/FoundationExtensions/DispatchTimeInterval+Extensions.swift

```diff
@@ -12,6 +12,8 @@
 //  Created by Nacho Soto on 12/2/21.
 
 // swiftlint:disable identifier_name
+import Foundation
+
```

Comment:
missing `import Foundation`s were causing the SPM issues

Reply:
Good catch. I wish Xcode was consistent with implicit imports.